README.md

```diff
@@ -98,11 +156,16 @@ npm run dev
 
 Head to [http://localhost:8000](http://localhost:8000), if a `build` folder is present in `frontend` directory then you will see the landing page of the project.
 
+**For more details related to installation and requirements please check the README in Backend directory.** 
+
+## Project Details
 
+Name : [ASHUTOSH KUMAR SINGH](https://github.com/lelouchB)
+Project Link : https://finalspaceapi.com/
+API Docs : https://finalspaceapi.com/docs/
 
```

Comment:
I have seen a lot of projects having Readme that features their contributors avatar automatically. 

Are you following what I am referring too?
😄

Maybe you can add it here.🚀

Reply:
Ok, I will look into this

Reply:
It should be quite simple, maybe it will be something that i need to add to my account. 

Just explore it a bit and let me know.

P.S. Work on other changes, I guess I will merge this sometime tommorow.
 
Thanks

Reply:
We need to use a bot for that as Github does not provide this feature or we need to manually update the README every few days
https://allcontributors.org/docs/en/bot/installation

Reply:
I think it is quite tedious 😅😅

Reply:
Yes, seems so
You can leave this comment then.
I will look into it
Thanks.